Index.html

```diff
@@ -20,358 +20,663 @@
     <link rel="stylesheet" href="style.css" />
   </head>
 
-  <body>
-    <header class="header-font">
-      <nav>
-        <a href="#">
-          <!-- <img src="" alt="Binabox" /> -->
-        </a>
-        <ul class="navigation">
-          <li><a href="#">Home</a></li>
-          <li><a href="#">Blog</a></li>
-          <li><a href="#">Catalog</a></li>
-          <li><a href="#">Contact Us</a></li>
+  <body class="body__homepage">
+    <header class="site-header">
```

Comment:
"Try to avoid excessively long class names. For instance, "site-header" can be simplified to "header", as it is understood by default that this header belongs to the website."